### PR TITLE
feat: modify Gemfile tamper check

### PR DESCRIPTION
**Pull request by prestonlimlianjie**
### Problem
We have migrated a bunch of Isomer repos onto Jekyll 4.0 - what this means is that we have:
- modified their current `staging` branch code to now use Jekyll 4.0
- modified the Netlify build command to use the `isomer-build` command that checks if their Gemfile is using `isomer-jekyll`.

However, several of these repos have not been active, and their admin users have yet to push the changes on staging branch to production. As such, code on the `master` branch still uses the old Gemfile (with `github-pages` instead of `isomer-jekyll`). And thus, we are unable to rebuild these websites on Netlify.

The main issue being unable to rebuild these sites is that they cannot inherit fixes from the main template. For example, we have recently pushed out a new government masthead for websites - but these websites will not have the new masthead until their admin users push changes to production.

### Solution
We had two proposals to fix this problem:

1. Manually go into Netlify and modify the build command for these sites to use the old build command. Trigger a rebuild. And then switch the build command back to the new build command.
2. Or modify the `isomer-build` Gemfile tamper check to be less strict - at least for now - but still check that our sites can only either use the new Gemfile (`isomer-jekyll`) or the existing one (`github-pages`). This will be in place until all sites have been migrated to Jekyll 4.0, or moved off of Isomer as a platform.

We decided to go with option 2 because it was less manual than option 1. And it would also allow us to continue to make updates to the template without having to go through the manual process of option 1 every single time.

### CODE REVIEW

File: overrides/Gemfile-github-pages [hunk 1]
```diff
@@ -0,0 +1,3 @@
+source "https://rubygems.org"
+
+gem "github-pages", group: :jekyll_plugins
```
Comment (alexanderleegs) [inline overrides/Gemfile-github-pages:3]: We should use single quotes for this file actually - we seem to consistently use single quotes for the github-pages gem, see:
https://github.com/isomerpages/ndp2020/blob/staging/Gemfile

**Review by alexanderleegs — CHANGES_REQUESTED** [review]

File: overrides/Gemfile-github-pages (hunk 1 above)
Comment (prestonlimlianjie) [reply overrides/Gemfile-github-pages:3]: oh dear

Comment (prestonlimlianjie) [reply overrides/Gemfile-github-pages]: good catch - fixed in d8c1ea5

**Review by alexanderleegs — APPROVED** [review]
lgtm!

**Pull request merged**